website/_dogfooding/_docs tests/toc-partials/_partial-with-props.mdx

```diff
@@ -0,0 +1,5 @@
+## This is a partial markdown with props
+
+### Props in headings dont work: {props.heading}
```

Comment:
Don't worry about that edge case, we don't support expressions in headings even in non-partial files atm. If we work on this it will be in another PR.